.github/workflows/deploy_all.yml

```diff
@@ -54,4 +54,26 @@ jobs:
         AWS_REGION: 'us-west-1'
         SOURCE_DIR: 'build/${{ matrix.target }}/_metadata/'
         DEST_DIR: 'Firmware/${{ env.version }}/${{ matrix.target }}/'
-
+  generic_metadata:
+    runs-on: ubuntu-latest
+    container: px4io/px4-dev-base-focal:2021-09-08
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        token: ${{secrets.ACCESS_TOKEN}}
+    - name: Build generic metadata
+      run: |
+        make airframe_metadata parameters_metadata
+        cd build/px4_sitl_default/docs
+        mkdir _metadata || true
+        cp parameters.xml, parameters.json.xz, airframes.xml _metadata
```

Comment:
I'm wondering if `build/px4_sitl_default/events` are also worth uploading to S3 (wasn't the case in the past).